daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc/Test.hs

```diff
@@ -32,9 +35,11 @@ import System.FilePath
 import qualified Text.XML.Light as XML
 
 
+newtype ColorTestResults = ColorTestResults{color :: Bool}
```

Comment:
I'd prefer if the field was called `getColorTestResults` or `unColorTestResults`.

Reply:
Cool looks good to me too, will make the change